package.json

```diff
@@ -300,31 +300,31 @@
   },
   "dependencies": {
     "archiver": "^2.1.1",
-    "cs-jsforce-metadata-tools": "^1.2.10",
     "elegant-spinner": "^1.0.1",
     "fs-extra": "^5.0.0",
-    "globule": "^1.2.0",
-    "jsforce": "github:celador/jsforce",
+    "globule": "^1.2.1",
+    "jsforce": "^1.8.4",
+    "jsforce-metadata-tools": "^1.3.0",
     "jszip": "^3.1.5",
     "klaw": "^2.1.1",
-    "lodash": "^4.17.5",
-    "moment": "^2.21.0",
-    "node-fetch": "^2.1.1",
+    "lodash": "^4.17.10",
+    "moment": "^2.22.2",
+    "node-fetch": "^2.1.2",
     "node-zip": "^1.1.1",
     "os": "^0.1.1",
-    "salesforcedx": "^42.5.0",
+    "salesforcedx": "^43.6.0",
     "sequin": "^0.1.1",
-    "typescript": "^2.5.3",
-    "unzip": "^0.1.11",
-    "vscode": "^1.1.11",
+    "typescript": "^2.9.2",
+    "unzip-stream": "^0.3.0",
```

Comment:
Why was this changed?  This can probably removed completely as I don't see it used in the current repository anywhere